src/stylesheets/components/_auth-alert.scss

```diff
@@ -1,28 +1,44 @@
 .auth-alert {
-  backdrop-filter: blur(rem(54px));
-  background: map-get($brand-gray, 'primary');
-  border-radius: rem(12px);
-  margin-bottom: rem(24px);
+  @extend .row;
```

Comment:
Do not use extend instead use the [mixins](https://github.com/twbs/bootstrap/blob/main/scss/mixins/_grid.scss) like `make-row()`, `make-container`, `make-col()`

@extend has a huge downside as it creates super long selectors (read [more](https://webinista.com/updates/dont-use-extend-sass/)). Mixins FTW.

Reply:
Updated the `extends` with `mixins` in [46261fd](https://github.com/rafayet-monon/surveyor/pull/13/commits/46261fd3c8960be1b9f2eada091395864d4a4b45) 🚀